exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterTest.java

```diff
@@ -244,4 +250,28 @@ void stringRepresentation() {
               "ZipkinSpanExporter{endpoint=http://zipkin:9411/api/v2/spans, compressionEnabled=false, readTimeoutMillis=15000}");
     }
   }
+
+  @Test
+  void testSuppressInstrumentation() {
+    AtomicBoolean suppressInstrumentation =
+        new AtomicBoolean(InstrumentationUtil.shouldSuppressInstrumentation(Context.current()));
+
+    assertFalse(suppressInstrumentation.get());
+
+    InstrumentationUtil.suppressInstrumentation(
+        () -> {
+          try (BytesMessageSender sender =
+              OkHttpSender.newBuilder()
+                  .endpoint("https://localhost")
+                  .encoding(Encoding.PROTO3)
+                  .build()) {
+            sender.send(Collections.singletonList(new byte[0]));
```

Comment:
I don't understand how this test is exercising the code added in this PR.

Reply:
I updated the test.